exporter.go

```diff
@@ -62,7 +63,7 @@ func (b *Exporter) Listen(e <-chan Events) {
 	for {
 		select {
 		case <-removeStaleMetricsTicker.C:
-			b.registry.removeStaleMetrics()
+			b.registry.RemoveStaleMetrics()
```

Comment:
could this be part of a maintenance routine internal to the expiring registry? That way we would not have to rely on the library users to do it regularly.